### DISCUSSION TITLE
Keep Most Recently-Used Account on Quick Reblog

### DISCUSSION THREAD

**Discussion by RadRascal** [AprilSylph/XKit-Rewritten#317] [Suggestion Box]
The (old) New XKit, when using Quick Reblog on a Tumblr account with sideblogs, used to leave the user on the one you used for your most recent Quick Reblog. The Quick Reblog on XKit Rewritten, unfortunately, does not do the same. Everytime I hover over the Reblog to use the Quick Reblog tool, it resets back to my main blog, which is really frustrating and just a hassle to reclick the blog I post on. If people were to have an issue with setting this up, maybe at least make it an option under the dropdown option menu for Quick Reblogs.

**Comment by AprilSylph** [comment]
This is already an option: "Remember the last selected blog in the popup". It can't persist across hard refreshes, though, because XKit Rewritten is set up to be account-neutral in order to play nicely with container tabs.
